Interlace/lib/core/input.py

```diff
@@ -76,7 +76,7 @@ def _pre_process_commands(command_list, task_name=None, is_global_task=True):
         blocker = None
         for command in command_list:
             command = str(command).strip()
-            if len(command) == 0:
+            if not command:
```

Comment:
If command variable contains either `0` or  `False` or `empty string ''`, will cause if block to be executed.
If command variable contains any value other than `0`, `False` and `empty string ''`, then if block wont execute
```py
>>> len("abcd") == 0 #assume abcd as command variable
False
>>> not "abcd"
False
```